rpm/signon-qt5.spec

```diff
@@ -32,13 +32,13 @@ BuildRequires: pkgconfig(libproxy-1.0)
 BuildRequires: pkgconfig(qt5-boostable)
 BuildRequires: fdupes
 Requires: mapplauncherd-qt5
-Obsoletes: signon
 
 %description
 %{summary}.
 
 %files
 %defattr(-,root,root,-)
+%license COPYING
```

Comment:
This is already on line 57 :thinking:

Reply:
Duh, noticed libsignon-qt5 missing license and got again confused when this one had license as last thing instead of being like this in the beginning.

Oh well, the other .spec file changes would be good and also here moving it first. Maybe I could this time have the license file on both to avoid confusion the next time. Wonder why the separate sub-package even exists, though.

Reply:
Updated like that.